lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -11,7 +11,7 @@
     name: vmware_vm_inventory
     plugin_type: inventory
     short_description: VMware Guest inventory source
-    version_added: "2.6"
+    version_added: 2.7
```

Comment:
don't remove the quotes, otherwise yaml parses it as a number, we want string

Reply:
OK.